src/components/board/board.test.js

```diff
@@ -1,8 +1,37 @@
+const board = require('../../services/boardService');
```

Comment:
Sense this is board.test.js - import board.js rather than boardServce.js so these test run against that file. You should also make a boardService.test.js and have a test for makeBoard().